airbyte-webapp/src/locales/en.json

```
@@ -495,6 +495,7 @@
   "docs.notFoundError": "We were not able to receive docs. Please click the link above to open docs on our website",
   "errorView.notFound": "Resource not found.",
   "errorView.notAuthorized": "You don’t have permission to access this page.",
+  "errorView.title": "Oops! Something went wrong...",
```

:keyboard: We're trying to use the utf8 ellipsis sign when using an ellipsis (instead of an written out dot dot dot).

```suggestion
  "errorView.title": "Oops! Something went wrong…",
```